### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by Ilithy**
Solves #1 

Here is the translation of the application interface
in French.
 * Translation of the *Strings.xml* file (in the `*/app/src/main/res/values` folder) to the *Strings.xml* file created in the `*/app/src/main/res/values-fr-rFR` folder
 * Adding the line `<string name="fr" translatable="false">Français</string>` in the *Strings.xml* file in the `*/app/src/main/res/values` folder

If any changes are needed, please let me know.

### CODE REVIEW

File: app/src/main/res/values-fr-rFR/strings.xml [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+
+    <string name="learn_sudoku_rules">Règles du Sudoku</string>
+
+    <string name="sudoku_rules_mistakes">Il y a une erreur ici, pouvez-vous la repérer? Peut-être même deux</string>
+    <string name="sudoku_rules_mistakes_highlight">Mettre en évidence les erreurs</string>
+    <string name="sudoku_rules_mistakes_explanation">La première erreur est deux 2 dans le sixième bloc. Vous souvenez-vous de la règle selon laquelle il ne peut pas y avoir de numéros en double dans un bloc? La deuxième erreur est le chiffre 6 en ligne 2, colonne 8 (R2C8) car il y a déjà un 6 dans cette colonne en R5C8 (Row 5 Column 8)</string>
+    <string name="naked_pairs_title">Paires nues</string>
+    <string name="naked_pairs_explanation">Il y a deux cellules qui partagent les deux mêmes marques. Pour le moment, nous ne pouvons pas dire lequel est le 2 et lequel est le 3\nMais parce que les règles du Sudoku nous disent que chaque bloc n\'a qu\'un seul "2" et un seul "3", nous pouvons supposer que l\'un d'eux est le 2 et l\'autre est le 3</string>
```
Comment (kaajjo) [inline app/src/main/res/values-fr-rFR/strings.xml:190]: You forgot to escape the apostrophe with `\` in `d'eux`
```suggestion
<string name="naked_pairs_explanation">Il y a deux cellules qui partagent les deux mêmes marques. Pour le moment, nous ne pouvons pas dire lequel est le 2 et lequel est le 3\nMais parce que les règles du Sudoku nous disent que chaque bloc n\'a qu\'un seul "2" et un seul "3", nous pouvons supposer que l\'un d\'eux est le 2 et l\'autre est le 3</string>
```

File: app/src/main/res/values-fr-rFR/strings.xml [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+    <string name="dialog_new_game_positive">Démarrer une nouvelle partie</string>
+    <string name="action_cancel">Annuler</string>
+    <string name="action_confirm">Oui</string>
+    <string name="action_play">Jouer</string>
+    <string name="action_continue">Continuer</string>
+    <string name="action_delete">Supprimer</string>
+
+    <string name="difficulty_unspecified">Non spécifié</string>
+    <string name="difficulty_simple">Simple</string>
+    <string name="difficulty_easy">facile</string>
```
Comment (kaajjo) [inline app/src/main/res/values-fr-rFR/strings.xml:18]: Should it start with lowercase?

**Comment by kaajjo** [comment]
Could you also add `<locale android:name="fr"/>` to `locales_config.xml` located in res/xml folder. So that users can select a language for the app in the system settings (Android 13) and via the picker in the app settings

File: app/src/main/res/values-fr-rFR/strings.xml (hunk 2 above)
Comment (Ilithy) [reply app/src/main/res/values-fr-rFR/strings.xml]: No, it's a mistake, thanks for pointing it out, it's corrected

**Comment by kaajjo** [comment]
Yep. Looks good. Thanks for your work 👍

**Pull request merged**